### PR TITLE
add-verbosity-flag - Add a verbosity flag so that kaniko isn't so noisy all the time

### PR DESCRIPTION
**Pull request by bwalding**
* Adds a verbosity flag and removes the default debug level (info is default built into kaniko command)
* Validates the flag to prevent any injection issues

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -65,7 +69,7 @@ runs:
       uses: docker://public.ecr.aws/l7o7z1g8/actions/kaniko-action:0.0.8
       with:
         entrypoint: kaniko-action
-        args: --dockerfile "${{ inputs.dockerfile }}" --context "${{ inputs.context }}" --destination "${{ inputs.destination }}" --registry-mirrors "${{ inputs.registry-mirrors }}" --skip-default-registry-fallback="${{ inputs.skip-default-registry-fallback }}"
+        args: --dockerfile "${{ inputs.dockerfile }}" --context "${{ inputs.context }}" --destination "${{ inputs.destination }}" --registry-mirrors "${{ inputs.registry-mirrors }}" --skip-default-registry-fallback="${{ inputs.skip-default-registry-fallback }}"  --verbosity "${{ inputs.verbosity }}"
```
Comment (pinaki124) [inline action.yml:72]: It appears there are two spaces before the verbosity flag.

**Review by pinaki124 — CHANGES_REQUESTED** [review]
You also need to bump the image version in the [workflow](https://github.com/cloudbees-io/kaniko/blob/main/.cloudbees/workflows/workflow.yaml#L30)

**Comment by bwalding** [comment]
> You also need to bump the image version in the [workflow](https://github.com/cloudbees-io/kaniko/blob/main/.cloudbees/workflows/workflow.yaml#L30)

I didn't spot that in the middle of a line - would it be better broken out as an env 
`ACTION_VERSION` and then ${ACTION_VERSION} in the line?  Or no need?

**Review by pinaki124 — APPROVED** [review]

File: action.yml [hunk 2]
```diff
@@ -32,6 +32,10 @@ inputs:
     description: >
       If set, fails build if registry-mirrors cannot pull image. If registry-mirrors is empty, this flag is ignored.
       Type: Boolean
+  verbosity:
+    default: debug
```
Comment (mgoltzsche) [inline action.yml:36]: Wouldn't it make sense to let it default to `verbosity=info`?

Comment (bwalding) [reply action.yml:36]: Yeah - I tend to agree that `info` is a better default. But I didn't want to change the behaviour as I'm an outsider :)

Comment (pinaki124) [reply action.yml:36]: I think it's ok to change it 😊

**Review by pinaki124 — APPROVED** [review]

File: action.yml (hunk 2 above)
Comment (bwalding) [reply action.yml]: https://github.com/cloudbees-io/kaniko/pull/11/commits/3dd2297ad6804aefbd7224fc4bdb668c3542a839 changes it to `info` as default.

**Comment by bwalding** [comment]
@pinaki124 @mgoltzsche What's the next step in getting this out?  (sure I can merge it, but I can merge pretty much anything in this place - doesn't mean it's a good idea!)

**Pull request merged**